README.md

```diff
@@ -40,44 +40,49 @@ Here is an example of how to use this package:
 package main
 
 import (
-        "log"
+	"log"
 
-        "github.com/mongo-go/lock"
+	"github.com/smartinov/mongo-lock/lock"
```

Comment:
Fix this and I'll merge it

Reply:
should be set to the previous value now "github.com/mongo-go/lock"